cubist/src/text.h

```diff
@@ -84,10 +84,10 @@
   "\t-C <members>\tcommittee model\n"                                          \
   "\t-h\t\tprint this message\n"
 #define T_TargetAtt "\n    Target attribute `%s'\n"
-#define TX_ReadData(c, a, f)                                                   \
-  "\nRead %d cases (%d attributes) from"                                       \
-  " %s.data\n",                                                                \
-      c, a, f
+#define TX_ReadData(c, a)                                                   \
+  "\nRead %d cases (%d attributes)"                                       \
+  "\n",                                                                \
+      c, a
```

Comment:
Being nitpicky here, but is there a reason for the "\" to be "right-aligned"?

Reply:
made into a one-liner